charms/trusty/clearwater-homestead/actions/create-update-user

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -ex
+
+# Get the configuration and action parameters.
+home_domain=$(config-get zone)
+number=$(action-get number)
+password=$(action-get password)
+
+
+# If the user doesn't exist, try to create them.  Otherwise, try to update them.
+if ! sudo -S /usr/share/clearwater/bin/display_user $number $home_domain ; 
```

Comment:
Why `sudo -S` (specifically the `-S`)?  We're not providing a username or password here.  Surely if the current user is not already authenticated, there's no way we can succeed?